### PR TITLE
Implement message functionality with sockets, and cleanup/refactor signup.

### PR DESCRIPTION
**Pull request by creativediego**

### CODE REVIEW

**Review by spencersolomon6 — APPROVED** [review]
Great work!

File: src/redux/tuitSlice.js [hunk 1]
```diff
@@ -6,14 +6,6 @@ import {
 } from '../services/tuits-service';
 import { dataOrStateError } from './helpers';
 
-/**
```
Comment (sabrasisler) [inline src/redux/tuitSlice.js:9]: Is this code being used? If not can it be deleted?

File: src/services/messages-service.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import axios from 'axios';
```
Comment (sabrasisler) [inline src/services/messages-service.js:2]: Can we add documentation to this file?

File: src/services/socket-config.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const SOCKET_URL = process.env.REACT_APP_SOCKET_URL;
+const CLIENT_URL = process.env.CLIENT_URL;
+
+export const socket = io(`${SOCKET_URL}`, {
+  cors: {
+    origin: CLIENT_URL,
+  },
+});
+
+// const createSocket = () => {
```
Comment (sabrasisler) [inline src/services/socket-config.js:11]: Is this commented code being used? Can it be deleted?

File: src/services/users-service.js [hunk 4]
```diff
@@ -19,9 +20,16 @@ export const updateUser = (user) =>
     .catch((err) => err.response.data);
 
 export const findAllUsers = () =>
```
Comment (sabrasisler) [inline src/services/users-service.js:28]: Can we add documentation for the specific services?

File: src/views/TuiterView/DoubleColumn.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (sabrasisler) [inline src/views/TuiterView/DoubleColumn.js]: What is this file being used for? What page is using this double view?

File: src/views/TuiterView/SingleColumn.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (sabrasisler) [inline src/views/TuiterView/SingleColumn.js]: What is this being used for? Please add documentation

**Review by sabrasisler — CHANGES_REQUESTED** [review]

**Pull request merged**